lib/testScenario/postmanCollectionRunnerClient.ts

```diff
@@ -687,7 +698,7 @@ export class PostmanCollectionRunnerClient implements TestScenarioRunnerClient {
           exec: this.postmanTestScript.generateScript({
             name: "AAD auth should be successful",
             types: ["ResponseDataAssertion", "OverwriteVariables"],
-            variables: new Map<string, string>([["bearerToken", "access_token"]]),
+            variables: new Map<string, string>([["bearerToken", "/access_token"]]),
```

Comment:
Is this intended change?

Reply:
Yes. We use jsonPointer to get response value.